src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindOperation.java

```diff
@@ -25,7 +25,11 @@
 public record FindOperation(
     CommandContext commandContext,
     List<DBFilterBase> filters,
-    DocumentProjector projection,
+    /**
+     * Projection used on input document; if no changes desired, identity projection. Only used for
+     * "pure" read operations: for updates (like {@code findOneAndUpdate}) is passed differently.
+     */
+    DocumentProjector inputProjection,
```

Comment:
Renamed here to reflect different application

Reply:
This will be result projection for Find and FindOne, think resultProjection sounds better in this case too.

Reply:
I guess that is true. Will change as suggested.